src/layout/CollapsibleList.js

```diff
@@ -0,0 +1,112 @@
+    CollapsibleList.prototype.publish("titleBackgroundColor", "#333333", "html-color", "Title background color",null,{tags:["Basic"]});
+    
+    CollapsibleList.prototype.publish("closedByDefault", false, "boolean", "Open by default if true",null,{tags:["Basic"]});
+
+    CollapsibleList.prototype.testData = function () {
+        this.pushListItem(new CollapsibleList().title("Form 1").testData2());
+        this.pushListItem(new CollapsibleList().title("Form 2").testData2());
+        return this;
+    };
+    CollapsibleList.prototype.testData2 = function () {
+        this.pushListItem(new Form().testData().showSubmit(false));
+        return this;
+    };
+    
+    CollapsibleList.prototype.pushListItem = function (widget) {
+        var contentArr = this.content();
+        contentArr.push(widget);
+        this.content(contentArr);
+        return this;
+    };
+
+    CollapsibleList.prototype.collapseClick = function (element) {
+        if(element.classed("closed")){
+            this._isClosed = false;
+            element.classed("open",true);
+            element.classed("closed",false);
+        } else {
+            this._isClosed = true;
+            element.classed("open",false);
+            element.classed("closed",true);
+        }
+    };
+
+    CollapsibleList.prototype.enter = function (domNode, element) {
+        HTMLWidget.prototype.enter.apply(this, arguments);
+        var context = this;
+        this._isClosed = this.closedByDefault();
+        element.classed(this._isClosed ? "closed" : "open",true);
+        
+        this.titleSpan = element.append("span").classed("collapsible-title",true);
+        this.iconDiv = element.append("div").classed("collapsible-icon",true);
+        this.ul = element.append("ul");
+        
+        this.icon = new FAChar().target(this.iconDiv.node());
+        
+        this.iconDiv.on("click",function(){
+            context.collapseClick(element);
+            context.render();
+        });
+        this.titleSpan.on("click",function(){
+            context.collapseClick(element);
+            context.render();
+        });
+    };
+
+    CollapsibleList.prototype.update = function (domNode, element) {
+        HTMLWidget.prototype.update.apply(this, arguments);
+        var context = this;
+        this.titleSpan.text(context.title().length > 0 ? context.title() : "CollapsibleList ["+context._id+"]");
+        var rows = this.ul.selectAll("li").data(this.content(),function(d) { return d._id; });
+        rows.enter()
+            .append(function(widget) {
+                var li = document.createElement("li");
+                if(widget._target === null){
+                    widget.target(li);
+                } else {
+                    return widget._target;
+                }
+                return li;
+            })
+        ;
+        rows.exit().remove();
+        rows.each(function(widget){
+            widget.render();
+        });
```

Comment:
@jbrundage If your working off of latest sources I think lines 97->99 are not needed.